corehq/apps/users/models.py

```diff
@@ -2445,3 +2451,9 @@ def can_edit_locations(self):
 
     def can_edit_web_users(self):
         return False
+
+
+class DjangoUserCommCareFields(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+
+    couch_user_id = models.CharField(max_length=255)
```

Comment:
decided not to index because i don't think there's a real use case for looking up the django user by couch id since we store the django id in the couch model